content/documentation/explanations/dispatching.md

```diff
@@ -252,3 +256,66 @@ def foo = store.get("foo");
 def bar = store.put("bar", "barValue");
 store.delete("baz");
 ```
+
```

Comment:
It looks like you removed the Groovy examples for:
* Calling an external API
* Use persistent storage (partially merged in "Analyse JSON body payload" example above that was also trimmed)

Can you set them back please?